### PR TITLE
Rw improve ux

### PR DESCRIPTION
**Pull request by rwilliams659**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [X] Refactoring

### Detailed Description
-I added "Back Home" buttons beneath each of the input forms in the html
-I then refactored our event listeners to be on parent containers - one for the buttons on the main page, and another that encompasses the whole form section (across all 3 forms), eliminating unneeded query selectors as part of this
-I then hooked up the "Back home" buttons to the BacktoMainPage function

**NOTE**:  It looks like I changed a ton on index.html but I really only added a few buttons and encased all of the form html in an additional section tag, which pushed down the rest of the content. 

### Why is this change required? What problem does it solve?
This improves the UX for the forms and makes use of event delegation to eliminate unnecessary global query selectors & event listeners. 

### Were there any challenges that arose while implementing this feature? If so, how were the addressed?
None!

### Files modified:
- [X] index.html
- [X] style.scss
- [X] scripts.js

### Relevant tickets:
Ticket #80 & #81

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -3,34 +3,46 @@ import './images/runner.jpg';
...
-hydrationDataButton.addEventListener('click', accessHydrationInputForm)
-submitHydrationButton.addEventListener('click', submitButton)
-submitSleepButton.addEventListener('click', submitButton)
-submitActivityButton.addEventListener('click', submitButton)
+mainPageButtons.addEventListener('click', mainPageClickAnalyzer);
+dataFormSection.addEventListener('click', formClickAnalyzer);
 
 window.onload = getData();
 
+function mainPageClickAnalyzer(event) {
```
Comment (jkwest-93) [inline src/scripts.js:25]: Nice work getting these event listeners on the parent container. This looks really clean!

**Comment by jkwest-93** [comment]
This looks really great! Nice job with the styling & functionality.
Screen Shot 2020-07-27 at 9 01 59 PM

**Comment by rwilliams659** [comment]
@jkwest-93 I did revert "today" back! Sorry if that was confusing. I wasn't sure if I should leave it with the hardcoded date or not.

**Comment by jkwest-93** [comment]
Thanks so much for clarifying & for changing the date back!! I'll switch it up in my local if I need to load that data.

**Comment by krogowsk531** [comment]
Great job with refactoring and adding the extra buttons!

**Pull request merged**